src/components/HotTags/index.tsx

```diff
@@ -30,7 +30,7 @@ export default function HotTags() {
 
   return (
     <div id="hot-tags" className="col-span-1 mb-8">
-      <SideCard.Header title="Hot tags">{/**<DropDown.TagsTimeframe type="text" />*/}</SideCard.Header>
+      <SideCard.Header title="Hot Tags">{/**<DropDown.TagsTimeframe type="text" />*/}</SideCard.Header>
```

Comment:
```suggestion
      <SideCard.Header title="Hot Tags" />
```
Since you are working on this, I suggest cleaning up the comments from legacy code.